source/includes/usage-examples/index-code-examples.c

```diff
@@ -1,20 +1,16 @@
+#include <stdio.h>
 #include <bson/bson.h>
 #include <mongoc/mongoc.h>
-#include <stdio.h>
 
 int
-main (int argc, char *argv[])
+main (void)
 {
     mongoc_client_t *client;
     mongoc_collection_t *collection;
-    bson_error_t error;
-
     mongoc_init ();
 
-    client =
```

Comment:
FYI this might mess with some highlighted lines below, would double-check the index landing page

Reply:
Yep you're right - fixed!